### DISCUSSION TITLE
26AE2F6D-406F-ACE1-617F-9C3364263CCC

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#88] [General]
# 26AE2F6D-406F-ACE1-617F-9C3364263CCC

The Ring Nebula (also catalogued as Messier 57, and NGC 6720) is a planetary nebula in the mildly northern constellation of Lyra around 2,567±115 light-years away. Such a nebula is formed when a star, during the last stages of its evolution before becoming a white dwarf, expels a vast luminous envelope of ionized gas into the surrounding interstellar space

https://celestia.mobi/addon?item=26AE2F6D-406F-ACE1-617F-9C3364263CCC&lang=en

**Comment by Pascualito1008** [comment]
Ok we need not Flex nebula SO into to the space have nebulas but he's formed random here is the milky way have to galatic Center is the sagittarius a* is the most black hole with more stars the stars from light
